lms/djangoapps/shoppingcart/tests/test_models.py

```diff
@@ -1144,7 +1144,7 @@ def test_donate_no_such_course(self):
         with self.assertRaises(CourseDoesNotExistException):
             Donation.add_to_order(self.cart, self.COST, course_id=fake_course_id)
 
-    def _assert_donation(self, donation, donation_type=None, course_id=None, unit_cost=None, line_desc=None):
+    def _assert_donation(self, donation, donation_type=None, course_id='', unit_cost=None, line_desc=None):
```

Comment:
This commit and the few above should really be squashed into the SubFieldBase removal commit.

Reply:
Sorry for being dense here, but why is the behavior changing so that empty course_ids are empty strings instead of `None`?

Reply:
@ormsbee Removal of the SubfieldBase metaclass is the change that started this behavior. We've modified the custom field as recommended by the Django docs by adding a `from_db_value` method. But the behavior has changed and we're admittedly a bit unsure about the ripple effects of the change.